DSESwift/Utils/ValueParser.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ValueParser.swift
+//  SensePlatform
+//
+//  Created by Tatsuya Kaneko on 30/10/15.
+//
+//
+
+import Foundation
+
+internal protocol BaseValueParser {
```

Comment:
internal keyword can (and should) be removed as it is the default. No need to make it explicit other than making it less readable.